CONTRIBUTING.md

```diff
@@ -26,13 +26,13 @@ This repository uses two primary tools for development:
 Install Nox in your local dev environment:
 
 ```console
-    $ pip install nox
```

Comment:
This proposed change takes the CONTRIBUTING.md doc out of alignment with the other docs. Please propose a format change at a higher level so it can be addressed across the documentation.